packages/typescript/internal/build_defs.bzl

```diff
@@ -252,13 +252,10 @@ def _ts_library_impl(ctx):
     ts_providers = compile_ts(
         ctx,
         is_library = True,
-        # Filter out the node_modules from deps passed to TypeScript compiler
-        # since they don't have the required providers.
-        # They were added to the action inputs for tsc_wrapped already.
-        # strict_deps checking currently skips node_modules.
+        # Strict_deps checking currently skips node_modules.
```

Comment:
I think we also should remove the corresponding workaround
https://github.com/bazelbuild/rules_typescript/blob/0f7c6f624c7ac56880d466ea81d83a57c72bf963/internal/build_defs.bzl#L102-L106
which I think means this TODO will also be fixed - node modules will only be action inputs when declared as deps

Reply:
Want to remove this whole comment at this point then or wait until the upstream change is done first?

Reply:
Either way, let's do it

Reply:
Ok. I'll send a PR to rules_typescript after I merge this